cpu/stressng_cpu.py.data/README

```diff
@@ -0,0 +1,32 @@
+ Description:
+-------------
+    - The main purpose of this script is to run CPU stress tests using the stress-ng program.
+
+ Input Parameters:
+-------------------
+    --runtime <time-in-seconds>
+    
+    The total runtime, if not specified, is defaulted to 7200 seconds i.e 2 hours.
+
+ Usage:
+--------
+    cpu_stress --runtime <time-in-seconds>
+    
+ List of stressors used - 
+--------------------------
+        - af-alg 
+        - bsearch 
+        - context 
+        - cpu
+        - crypt
+        - hsearch
+        - longjmp
+        - lsearch
+        - matrix
+        - qsort
+        - str
+        - stream
+        - tsearch
+        - vecmath
+        - wcs
+
```

Comment:
please remove this white space

Reply:
The reason for the whitespace is - later we get a "No Newline..." remark during the checks. Hence added a newline at the end. Please refer here : https://github.com/avocado-framework-tests/avocado-misc-tests/pull/2444#discussion_r1006830901

Reply:
Done.